wagtail/core/models.py

```diff
@@ -2816,6 +2821,10 @@ def get_task_states_user_can_moderate(self, user, **kwargs):
         else:
             return TaskState.objects.none()
 
+    @classmethod
+    def get_description(cls):
+        return _("Choose which Wagtail Groups can approve this task")
```

Comment:
I like the idea of a task description, but to me this reads more like help text/instruction for the `groups` field, unless I'm missing something. What about something like: "Members of chosen Wagtail Groups can approve this task", or "A task which can be approved by members of specific Wagtail Groups", or something along those lines?

Reply:
Good shout. Updated the wording!

Reply:
Great, thanks!